src/components/TodoError/TodoError.tsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useState } from 'react';
+import classNames from 'classnames';
+
+type TodoErrorProps = {
+  errorMsg: string
+};
+
+export const TodoError = ({ errorMsg }: TodoErrorProps) => {
+  const [hidden, setHidden] = useState(true);
+
+  useEffect(() => {
+    if (errorMsg) {
+      setHidden(false);
+    }
+  }, []);
```

Comment:
The dependency array in useEffect is empty, which means the effect will only run once after the initial render, and not on updates. It should include the errorMsg prop to ensure the error message is displayed whenever it changes.